Tests/Unit/Extend/Controller/Admin/PaymentMainTest.php

```diff
@@ -57,13 +57,43 @@ public function testRender($bValidationResult, $bExpected)
     public function renderProvider()
     {
         return [
-            'validation successful' => [true, false],
+            'validation success' => [true, true],
             'validation failure' => [false, true],
         ];
     }
 
     public function testSave()
     {
+        $_POST['editval']['oxpayments__allowed_currencies'] = [];
+
+        try {
+            $this->_controller->save();
+        } catch (\Exception $exc) {
```

Comment:
Why is this try/catch block needed? Shouldn't the test fail anyway if an exception is thrown?